cmd/internal/cmdprinter/netmap.go

```diff
@@ -1,4 +1,4 @@
-package common
+package cmdprinter
```

Comment:
why do we need that renaming? why not a separate commit?

Reply:
https://github.com/nspcc-dev/neofs-node/pull/2529#discussion_r1308887656 and https://github.com/nspcc-dev/neofs-node/pull/2529#discussion_r1307794356 and it nearly only connected with netmap-candidates diff

Reply:
so import cycles or what exactly made you move that file? not critical but just was surprised it is a required change

Reply:
I wanted to use `PrettyPrintNodeInfo` in `neofs-adm` (netmap_candidates.go), so I could just copy this function, but with @roman-khimov we decided to move internal/common/netmap.go file from `neofs-cli` to avoid code repetition.